source/fundamentals/linq.txt

```diff
@@ -1060,3 +1060,14 @@ uses a scalar operation, then prints the translated query:
    ``LoggedStages`` is not thread-safe. Executing a query and accessing the
    associated ``LoggedStages`` property from multiple threads might have
    non-deterministic results.
+
+Additional Information
+----------------------
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+For a complete list of supported LINQ methods, see the following API Documentation:
```

Comment:
i: [the SG advises against both stacking headings and having a single heading at a level](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#:~:text=Avoid%20having%20only%20a%20single%20heading%20at%20any%20level)

```suggestion
API Documentation
----------------------

For a complete list of supported LINQ methods, see the following API Documentation:
```